packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -153,50 +143,45 @@ export default class extends EntityModel {
    * Get the user linked to the org membership.
    */
   async getUser(graphApi: GraphApi) {
-    const incomingOrgMembershipLinkEntityModels =
-      await LinkEntityModel.getByQuery(graphApi, {
-        all: [
-          {
-            equal: [
-              { path: ["rightEntity", "uuid"] },
-              { parameter: this.getEntityUuid() },
-            ],
-          },
-          {
-            equal: [
-              { path: ["leftEntity", "ownedById"] },
-              { parameter: this.getOwnedById() },
-            ],
-          },
-          {
-            equal: [
-              { path: ["type", "versionedUri"] },
-              {
-                parameter:
-                  SYSTEM_TYPES.linkEntityType.hasMembership.getSchema().$id,
-              },
-            ],
-          },
-          {
-            equal: [{ path: ["version"] }, { parameter: "latest" }],
-          },
-          {
-            equal: [{ path: ["archived"] }, { parameter: false }],
-          },
-        ],
-      });
+    const userEntityModels = await EntityModel.getByQuery(graphApi, {
```

Comment:
Similar to the prior comment, if this class where to extend `LinkEntityModel`, the pre-fetched `this.leftEntityModel` could be used here instead.

Reply:
Handled by 9fa54ada81cf0e59578d49550d729a9a1dcc90d2